### PR TITLE
Added Laracon EU draft

### PR DESCRIPTION
**Pull request by hannesvdvreken**
### Added

* Laracon EU blog post
* CSS class for responsive embeds of youtube videos

### CODE REVIEW

File: _drafts/laraconeu-2015.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+The speaker line-up was very strong again this year. Some speakers like Adam Wathan, Matt Stauffer, Ben Corlett, Jeffrey Way and Taylor Otwell
+even gave a talk at both Laracon US and Laracon EU.
+
+At madewithlove we use Laravel a lot, so a couple of us (Maxime, Bram and me) went to Amsterdam
+to take part in this event and mingle with the community.
+
+## Monday
+
+I didn't attend a workshop, though they all looked interesting. Would have loved to attend all of them, but one simply cannot attend multiple.
```
Comment (Anahkiasen) [inline _drafts/laraconeu-2015.md]: ONE DOES NOT SIMPLY WALK INTO WORKSHOP

File: _drafts/laraconeu-2015.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Two great resources for my talk were [phppackagechecklist](phppackagechecklist.com) and [Principles of Package Design](https://leanpub.com/principles-of-package-design/).
+The full slide deck for your reference:
+
+<script async class="speakerdeck-embed" data-id="122248791e7546bd81f60734948d6c74" data-ratio="1.33333333333333" src="//speakerdeck.com/assets/embed.js"></script>
+
+I'll also include the video once it's available.
+
+After my talk there was only one more by Jeffrey Way. His voice sounded very familiar to everyone at the conference,
+because he's the creator of Laracasts and the majority of screencasts on it. He talked on stage about Laravel's philosophy,
+how developer experience is more important than doing thing according to best practises and scalability.
```
Comment (Anahkiasen) [inline _drafts/laraconeu-2015.md]: things

Comment (Anahkiasen) [inline _drafts/laraconeu-2015.md]: practices

File: _drafts/laraconeu-2015.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The day was concluded with and audio-visual performance with a Kinekt, and a closing reception.
+
+Don't worry, Amsterdam, we'll be back.
```
Comment (Anahkiasen) [inline _drafts/laraconeu-2015.md]: Maybe put the name of the speakers in bold and/or link to their twitter so they stand out better in the text

File: _drafts/laraconeu-2015.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+* Document how it can be reproduced
+* Contact the people responsible for the leak
+
+Konstantin Kudryashov, better known as [@everzet](https://twitter.com/everzet), talked about software costs.
+Cost of introduction, cost of change, cost of ownership.
+[His slides](http://www.slideshare.net/everzet/minmaxing-software-costs-laracon-eu-2015) can tell you more than I can,
+because I was getting into my zone because I was up next.
+
+My talk was about Package Development. I talked about both beginner steps and advanced techniques for novice package developers.
+Two great resources for my talk were [phppackagechecklist](phppackagechecklist.com) and [Principles of Package Design](https://leanpub.com/principles-of-package-design/).
```
Comment (xavierbarbosa) [inline _drafts/laraconeu-2015.md]: Maybe better to change to [PHP Package Checklist](phppackagechecklist.com).

File: _drafts/laraconeu-2015.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+<script src="https://gist.github.com/hannesvdvreken/4aeb00405b7c36dbb08e.js"></script>
+
+After the first break Esther Kok, professor at the university of Nijmegen, talked about law for hackers. She explained
+how to keep your criminal record clean when you stumble upon a breach in security. A quick recap:
+
+* Don't tell anyone
+* Document how it can be reproduced
+* Contact the people responsible for the leak
+
+Konstantin Kudryashov, better known as [@everzet](https://twitter.com/everzet), talked about software costs.
```
Comment (Anahkiasen) [inline _drafts/laraconeu-2015.md]: Maybe put the name of the speakers in bold and/or link to their twitter so they stand out better in the tex

File: _drafts/laraconeu-2015.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+<script async class="speakerdeck-embed" data-id="122248791e7546bd81f60734948d6c74" data-ratio="1.33333333333333" src="//speakerdeck.com/assets/embed.js"></script>
+
+I'll also include the video once it's available.
+
+After my talk there was only one more by Jeffrey Way. His voice sounded very familiar to everyone at the conference,
+because he's the creator of Laracasts and the majority of screencasts on it. He talked on stage about Laravel's philosophy,
+how developer experience is more important than doing thing according to best practises and scalability.
+
+The day was concluded with and audio-visual performance with a Kinekt, and a closing reception.
```
Comment (xavierbarbosa) [inline _drafts/laraconeu-2015.md]: ...with an audio...

File: _drafts/laraconeu-2015.md (hunk 1 above)
Comment (hannesvdvreken) [inline _drafts/laraconeu-2015.md]: Hahaha :laughing: yeah this line sucks.

**Comment by Anahkiasen** [comment]
@hannesvdvreken Would it be worth adding a few lines on how Laracon differs from typical dev conferences? I've found It seems to become more and more focused on developers rather than developing, with very few actual technical talks and more "people" talks

**Comment by hannesvdvreken** [comment]
@Anahkiasen like this? https://github.com/madewithlove/blog/commit/289cfe7e173f9e5a103fe9bc2a02006f29e15a08

**Pull request merged**